src/override/rust.cc

```diff
@@ -36,3 +37,23 @@ rust_realloc(void* ptr, size_t alignment, size_t old_size, size_t new_size)
   }
   return p;
 }
+
+extern "C" SNMALLOC_EXPORT void* sn_malloc(size_t size)
+{
+  return snmalloc_malloc(size);
```

Comment:
`snmalloc_malloc` will already be `extern "C"` and have `SNMALLOC_EXPORT` applied to it.  So I am not sure why you have the level of indirection?

Reply:
Ah, that makes sense, I'll correct it now.